homework/index.js

```diff
@@ -30,21 +30,43 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const liRepository = createAndAppend('li', ul);
+    createAndAppend('b', liRepository, { text: ' Repository:  ' });
+    createAndAppend('a', liRepository, {
+      text: repo.name,
+      href: repo.html_url,
+      target: '_blank',
+    });
+    const liDescription = createAndAppend('li', ul);
+    createAndAppend('b', liDescription, { text: 'Description:  ' });
+    createAndAppend('span', liDescription, { text: repo.description });
+    const liForks = createAndAppend('li', ul);
+    createAndAppend('b', liForks, { text: 'Forks:  ' });
+    createAndAppend('span', liForks, { text: repo.forks });
+    const liUpdated = createAndAppend('li', ul);
```

Comment:
I see repetitive code here: create a `li`, add a label text child and add a text value child. You do that three times. Time for think of extracting this repetition into a function? (DRY principle).

Reply:
Thank you for feedback . i will improve those cases